### PR TITLE
Admin can set application status

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: https://www.pivotaltracker.com/story/show/133950603

_As an Admin
In order to get members into SHF and get their money
I need to be able to accept/reject their application_

Changes proposed in this pull request:
- Add status to Membership Application (default 'pending')
- Adds select box to Membership Application show page
- Sets select box at current status of Membership Application (to be refactored)
- Show timestamp and note of when it was accepted

### CODE REVIEW

File: app/views/membership_applications/show.html.haml [hunk 1]
```diff
@@ -8,3 +8,21 @@
...
+      - if @membership_application.status.to_i == 2
+        - chosen = 2
+      - elsif @membership_application.status.to_i == 3
+        - chosen = 3
+      - else
+        - chosen = 1
+      = f.select(:status, options_for_select([['Pending', 1],
+                                              ['Accepted', 2],
+                                              ['Rejected', 3]],
+                                              chosen))
```
Comment (patmbolger) [inline app/views/membership_applications/show.html.haml]: the `select` method will set the currently-selected option based on the current value of select field (:status) versus the select options collection.  So, lines 19-21 could be:

```
== f.select(:status, [['Pending', 'Pending'],
                      ['Accepted', 'Accepted'],
                      ['Rejected', 'Rejected']])
```

And lines 13-18 can be removed.

File: app/views/membership_applications/show.html.haml [hunk 2]
```diff
@@ -8,3 +8,21 @@
...
     %p= @membership_application.phone_number
   %p= mail_to @membership_application.company_email, @membership_application.company_email
+  .wpcf7
+    = form_for @membership_application, url: membership_application_path do |f|
+      - if @membership_application.status.to_i == 2
+        - chosen = 2
+      - elsif @membership_application.status.to_i == 3
+        - chosen = 3
+      - else
+        - chosen = 1
```
Comment (patmbolger) [inline app/views/membership_applications/show.html.haml]: `status` is implemented as a string (for the time being), so the above assignment expressions would not seem to work.

**Review by patmbolger — COMMENTED** [review]
Review the non-test parts of the PR - please see comments.

File: app/views/membership_applications/show.html.haml (hunk 1 above)
Comment (thesuss) [reply app/views/membership_applications/show.html.haml]: Aha!. Thanks so much.

**Comment by thesuss** [comment]
there we go. 19 files changed. Too many, but still quite a manageable PR. Finally. And thanks @patmbolger for the tip for the select box. 

Ready to be reviewed by @AgileVentures/shf-project-team

**Review by weedySeaDragon — APPROVED** [review]
We discussed this PR in a hangout.  It does allow the admin to change the status.  Agreed that it's time to close this as is and then open a separate PR to clean up and refactor as needed.

**Review by weedySeaDragon — APPROVED** [review]

**Review by diraulo — APPROVED** [review]
We'll take care of the CSS issues in a separate PR. I created an issue for that on [tracker](https://www.pivotaltracker.com/story/show/134212887)

**Pull request merged**